Packages/com.unity.inputsystem/InputSystem/Plugins/DualShock/DualShockGamepadHID.cs

```diff
@@ -77,6 +77,68 @@ public FourCC format
         }
     }
 
+    /// <summary>
+    /// Structure of HID input reports for PS3 DualShock 3 controllers.
+    /// </summary>
+    [StructLayout(LayoutKind.Explicit, Size = 32)]
+    public struct DualShock3HIDInputReport : IInputStateTypeInfo
```

Comment:
How different are the two? Any chance we can just have the same struct and use a layout variant?

Reply:
Nope. 100% different in binary layout.